package.json

```diff
@@ -7,9 +7,16 @@
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
     "@tippyjs/react": "^4.2.0",
+    "bounty": "^1.3.0",
     "react": "^17.0.1",
+    "react-confetti": "^6.0.0",
     "react-dom": "^17.0.1",
+    "react-dom-confetti": "^0.2.0",
+    "react-rewards": "^1.1.2",
     "react-scripts": "4.0.0",
+    "react-spring": "^8.0.27",
+    "styled-components": "^5.2.1",
```

Comment:
I guess these dependencies are not required?

Reply:
Done 💯